packages/fxa-auth-server/lib/senders/emails/templates/postChangeRecoveryPhone/index.mjml

```diff
@@ -0,0 +1,25 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="postChangeRecoveryPhone-title">You changed your backup recovery phone</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body">
+      <span data-l10n-id="postChangeRecoveryPhone-description">You now have a new backup recovery phone. Your previous phone number was deleted.</span>
+    </mj-text>
+    <mj-text css-class="text-body-no-margin">
+      <span data-l10n-id="postChangeRecoveryPhone-requested-device">You requested it from:</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+<%- include('/partials/userInfo/index.mjml') %>
+
+<%- include('/partials/automatedEmailResetPassword/index.mjml') %>
```

Comment:
Replace with `automatedEmailChangePassword`?

Reply:
It looked like in Figma, we used the "reset password" terminology for all of these, not change password.

Reply:
Yes, but I think it might have just been copied from past designs and not an intentionally decision.